contrib/lua/premake5.lua

```diff
@@ -21,8 +21,8 @@ project "lua-lib"
 		"etc/*.c"
 	}
 
-	configuration "linux or bsd or hurd or aix"
+	filter "system:linux or system:bsd or system:hurd or system:aix"
```

Comment:
Specifying the prefix on each item isn't required, `filter "system:linux or bsd or hurd or aix"` should work just fine. Personally, I prefer the less verbose way, but that is just a personal preference.